concepts/05 Widgets/Form/01 Getting Started with Form/40 Organize the Form Items/10 In Columns.md

```diff
@@ -0,0 +1,167 @@
+To keep the fixed number of columns, specify the number in the [colCount](/Documentation/ApiReference/UI_Widgets/dxForm/Configuration/#colCount) property. 
+
+The Form can also automatically adjust the number of columns to the screen width. Specify the [screenByWidth](/Documentation/ApiReference/UI_Widgets/dxForm/Configuration/#screenByWidth) and [colCountByScreen](/Documentation/ApiReference/UI_Widgets/dxForm/Configuration/colCountByScreen/) properties to enable this feature. 
```

Comment:
These two features are about the same thing and cannot be used together. It would be natural to describe these features by conterpointing them with each other. Something like:

```suggestion
The Form can organize items in a fixed number of columns or automatically adjust it to the screen width. To set the fixed number of columns, assign it to the [colCount](/Documentation/ApiReference/UI_Widgets/dxForm/Configuration/#colCount) property as done in the code below. If the number should be auto-adjusted, configure the [screenByWidth](/Documentation/ApiReference/UI_Widgets/dxForm/Configuration/#screenByWidth) and [colCountByScreen](/Documentation/ApiReference/UI_Widgets/dxForm/Configuration/colCountByScreen/) properties. 
```